docs/book/basics/simple_resource.md

```diff
@@ -58,8 +58,26 @@ used to configure code generators to run against this code.  The code generators
 generate boilerplate functions and types to complete the Resource definition.
 To learn more on configuring code generation see the *Code Generation* chapter.
 
-Note: The `// +kubebuilder:validation:Pattern=.+:.+` annotation declares Pattern validation
-requiring that the `Image` field match the regular expression `.+:.+`
+**validation**
+Format: `// +kubebuilder:validation:<key=value>`
+The validation annotation supports CRD validation ([OpenAPI v3 schema](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schemaObject)). The last `key-value` part should be a sigle key-value pair. If you have multiple validation key-value pairs, should put them into separte annotation comments.
+Currently, supporting keys are `Maximum`, `Minimum`, `MaxLength`, `MinLength`, `MaxItems`, `MinItems`, `UniqueItems`, `Enum`, `Pattern`, `ExclusiveMaximum`, `ExclusiveMinimum`, `MultipleOf`, `Format`
+The `// +kubebuilder:validation:Pattern=.+:.+` annotation declares Pattern validation requiring that the `Image` field match the regular expression `.+:.+`
+
+**subresource**
```

Comment:
I would suggest splitting `status` and `scale` sub-resources in two separate topics. `Status` is going to be used by almost all the resources, but `scale` is a bit special use-case.

Reply:
Good point!